hopr-socks/src/main.rs

```diff
@@ -0,0 +1,203 @@
+use anyhow::Context;
```

Comment:
The hopr-lib codebase uses `thiserror` though.

Reply:
Let's do this while doing the testing framework. With the tests being implemented, it will be clearer what errors can occur. Handling them will with custom errors will be done at this point.